test/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "#": "We must put jest in a sub-directory otherwise VS Code somehow picks up",
   "#": "the types and generates conflicts with mocha.",
```

Comment:
```suggestion
  "#": "We must put jest in a sub-directory otherwise VS Code somehow picks up",
  "#": "the types and generates conflicts with mocha.",
```